docs/sphinx/formats/ionpath-mibi.rst

```diff
@@ -0,0 +1,51 @@
+.. index:: Ionpath MIBI
+.. index:: .tif, .tiff
+
+Ionpath MIBI
+===============================================================================
+
+Extensions: .tif, .tiff
+
+Developer: `IonPath <https://www.ionpath.com/>`_
+
+Owner: `IonPath <https://www.ionpath.com/>`_
+
+**Support**
+
+
+BSD-licensed: |no|
+
+Export: |no|
+
+Officially Supported Versions: 0.1
+
+Reader: IonpathMIBITiffReader (:bfreader:`Source Code <IonpathMIBITiffReader.java>`, :doc:`Supported Metadata Fields </metadata/IonpathMIBITiffReader>`)
+
+
+
+
+We currently have:
+
+a few IonPath MIBI datasets
+
+We would like to have:
+
```

Comment:
There is nothing following the colon.  Should we say "nothing at this time" or "additional image files"?

Reply:
Good point. I would say `additional sample files and a public specification` would be the most appropriate here. Can push an additional commit.

Reply:
wfm